app/models/user.rb

```diff
@@ -5,4 +5,5 @@ class User < ApplicationRecord
   has_many :bookings, dependent: :destroy
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  validates :first_name, :last_name, :email, :password, presence: true
```

Comment:
uniqueness of email => validates :email, uniqueness: true

Reply:
agreed!